### PR TITLE
More stupid cat names

### PR DESCRIPTION
**Pull request by Sartorius7**
Adds a few more cat names to the list

### CODE REVIEW

File: strings/names/cats.txt [hunk 1]
```diff
@@ -2,13 +2,15 @@ names@=
 	Gary@,Mittens@,Mr. Jingles@,Rex@,Jasmine@,Litterbox
 	Reginald@,Poosycat@,Dr. Purrsworthy@,Lt. Scratches@,Michael Catson
 	Fluffy@,Mr. Purrfect@,Lord Furstooth@,Lion-O@,Johnathan@,Gary Catglitter
-	Chat de Gaulle@,Ratbag
+	Chat de Gaulle@,Ratbag@,Mitts@,Mr. Fluffington@,Meowxie@,Fluffy The Cat
```
Comment (Tarmunora) [inline strings/names/cats.txt:5]:
```suggestion
	Chat de Gaulle@,Ratbag@,Mitts@,Mr. Fluffington@,Meowxie
```
we already had `fluffy` above and I didn't notice when suggesting that one 😔

Comment (Sartorius7) [reply strings/names/cats.txt]: Fixed it.

**Review by Tarmunora — APPROVED** [review]

File: strings/names/cats.txt [hunk 2]
```diff
@@ -2,13 +2,15 @@ names@=
...
+	Prunella@,Poonella@,SEXCOPTER@,Fat, Lazy Piece of Shit@,Jones Mk. II@,Fish eater
+	Jones Mk. III@,Jones Mk. IV@,Jones Mk.V@,Mr. Meowgi@,Meoxie@,Zeta Jones@,Double Zeta Jones@,Nu Jones
+	Furrston von Purringsworth@,Garfadukecliff@,SyndiCat@,Rosa Fluffemberg@,Graxor The Litterbox Overlord
 	Karl Meowx@,Margaret Scratcher@,Marcel Purroust@,Franz Katka@,Das Katpital
 	Proletaricat@,Perestroikat@,Mewy P. Newton@,Fidel Catstro@,George Lucats
 	Lin Miao@,Felix Purrzhinsky@,Pol Pet@,Piggy@,Long Kitty@,Caterella@,Aristocat
 	Arthur Cat Clarke@,Prick@,Pantalaimon@,Purrsula K. Le Guin@,Douglas Pat 'Ems
 	Gato@,Zebra@,Possum@,Myst@,Too@,P@,Z@,Catopus
+	Jennyanydots@,Growltiger@,Rum Tum Tugger@,Jellicle Cat@,Mungojerrie@,Rumpleteazer@,Old Deuteronomy@,The Great Rumpus Cat
+	Mr. Mistoffelees@,Macavity@,Gus The Theatre Cat@,Asaparagus@,Bustopher Jones@,Skimbleshanks@,Cat Morgan
```
Comment (Sovexe) [inline strings/names/cats.txt:16]: Maybe I don't get the joke, but if not that's not how you spell Asparagus

Comment (Sartorius7) [reply strings/names/cats.txt:16]: D'oh!

**Review by Sovexe — APPROVED** [review]

**Comment by Kyle2143** [comment]
I removed the changelog here. PR's such as these don't warrant a changelog entry.

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**